com.sap.cloud.lm.sl.cf.persistence/src/test/java/com/sap/cloud/lm/sl/cf/persistence/services/DatabaseFileServiceTest.java

```diff
@@ -1,252 +1,313 @@
-        personalFiles = fileService.listFiles(MY_SPACE_ID, PERSONAL_NAMESPACE);
-        assertEquals(1, personalFiles.size());
-
-        systemFiles = fileService.listFiles(MY_SPACE_ID, SYSTEM_NAMESPACE);
-        assertEquals(1, systemFiles.size());
-
-        fileService.deleteFile(MY_SPACE_ID, systemFiles.get(0)
-            .getId());
-
-        personalFiles = fileService.listFiles(MY_SPACE_ID, PERSONAL_NAMESPACE);
-        assertEquals(1, personalFiles.size());
-
-        systemFiles = fileService.listFiles(MY_SPACE_ID, SYSTEM_NAMESPACE);
-        assertEquals(0, systemFiles.size());
-    }
-
-    @Test
-    public void testDeleteFile() throws FileStorageException {
-        fileService.deleteFile(MY_SPACE_ID, storedFile.getId());
-
-        FileEntry missingEntry = fileService.getFile(MY_SPACE_ID, storedFile.getId());
-        assertNull(missingEntry);
-
-        List<FileEntry> namespaceFiles = fileService.listFiles(MY_SPACE_ID, SYSTEM_NAMESPACE);
-        assertEquals(0, namespaceFiles.size());
-    }
-
-    @Test
-    public void testDeleteByModificationTime() throws Exception {
-        long currentMillis = System.currentTimeMillis();
-        final long oldFilesTtl = 1000 * 60 * 10; // 10min
-        Date pastMoment = new Date(currentMillis - 1000 * 60 * 15); // before 15min
-
-        FileEntry fileEntryToRemain1 = addFileEntry(MY_SPACE_ID);
-        FileEntry fileEntryToRemain2 = addFileEntry(MY_SPACE_2_ID);
-        FileEntry fileEntryToDelete1 = addFileEntry(MY_SPACE_ID);
-        FileEntry fileEntryToDelete2 = addFileEntry(MY_SPACE_2_ID);
-
-        setMofidicationDate(fileEntryToDelete1, pastMoment);
-        setMofidicationDate(fileEntryToDelete2, pastMoment);
-
-        int deletedFiles = fileService.deleteModifiedBefore(new Date(currentMillis - oldFilesTtl));
-
-        assertNotNull(fileService.getFile(MY_SPACE_ID, fileEntryToRemain1.getId()));
-        assertNotNull(fileService.getFile(MY_SPACE_2_ID, fileEntryToRemain2.getId()));
-
-        assertEquals(2, deletedFiles);
-        assertNull(fileService.getFile(MY_SPACE_ID, fileEntryToDelete1.getId()));
-        assertNull(fileService.getFile(MY_SPACE_2_ID, fileEntryToDelete2.getId()));
-    }
-
-    private void setMofidicationDate(FileEntry fileEntry, Date modificationDate) throws SQLException {
-        PreparedStatement statement = null;
-        try {
-            statement = testDataSource.getDataSource()
-                .getConnection()
-                .prepareStatement(MessageFormat.format(UPDATE_MODIFICATION_TIME, FileService.DEFAULT_TABLE_NAME));
-            statement.setTimestamp(1, new java.sql.Timestamp(modificationDate.getTime()));
-            statement.setString(2, fileEntry.getId());
-            statement.executeUpdate();
-        } finally {
-            JdbcUtil.closeQuietly(statement);
-        }
-    }
-
-}
+package com.sap.cloud.lm.sl.cf.persistence.services;

```

Comment:
Formatting? If not, check the whitespace character in your Eclipse

Reply:
I think it's fixed now.